src/ima-dai.js

```diff
@@ -328,9 +328,10 @@ class ImaDAI extends BasePlugin implements IAdsControllerProvider, IEngineDecora
   }
 
   _loadImaDAILib(): Promise<*> {
+    const protocol = /^(https?:)/i.test(document.location.protocol) ? document.location.protocol : 'https:';
     return (this._isImaDAILibLoaded()
       ? Promise.resolve()
-      : Utils.Dom.loadScriptAsync(this.config.debug ? ImaDAI.IMA_DAI_SDK_DEBUG_LIB_URL : ImaDAI.IMA_DAI_SDK_LIB_URL)
+      : Utils.Dom.loadScriptAsync(this.config.debug ? protocol + ImaDAI.IMA_DAI_SDK_DEBUG_LIB_URL : protocol + ImaDAI.IMA_DAI_SDK_LIB_URL)
```

Comment:
```suggestion
      : Utils.Dom.loadScriptAsync(protocol + this.config.debug ? ImaDAI.IMA_DAI_SDK_DEBUG_LIB_URL : ImaDAI.IMA_DAI_SDK_LIB_URL)
```

Reply:
No, it's if we are in debug mode and then I add the protocol

Reply:
```suggestion
      : Utils.Dom.loadScriptAsync(protocol + (this.config.debug ? ImaDAI.IMA_DAI_SDK_DEBUG_LIB_URL : ImaDAI.IMA_DAI_SDK_LIB_URL));
```

I had a mistake in the code suggestion, but what I meant is that you can save the double protocol concat.
And because this is getting so verbose I would separate the string calculation from the method call: 
```javascript
const imaSdkUrl = protocol + (this.config.debug ? ImaDAI.IMA_DAI_SDK_DEBUG_LIB_URL : ImaDAI.IMA_DAI_SDK_LIB_URL);
...
Utils.Dom.loadScriptAsync(imaSdkUrl);
```